clients/src/main/java/org/apache/kafka/clients/consumer/internals/NetworkClientDelegate.java

```diff
@@ -144,6 +145,18 @@ public void tryConnect(Node node) {
      * @param currentTimeMs current time
      */
     public void poll(final long timeoutMs, final long currentTimeMs) {
+        poll(timeoutMs, currentTimeMs, false);
+    }
+
+    /**
+     * Returns the responses of the sent requests. This method will try to send the unsent requests, poll for responses,
```

Comment:
nit: The poll methods do not return the responses of the sent requests. They return nothing at all. I think the rest of the comment apart from the first sentence is OK. The other poll method has a similar mistake.